dockerfiles/kubetools/kubectl-1.22.Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM docker.io/library/alpine:latest
+
+ARG VCS_REF=master
+ARG BUILD_DATE=""
+ARG REGISTRY_PATH=docker.io/paritytech
+ARG KUBE_VERSION="1.22.15"
```

Comment:
I would suggest to not include the version in the file name. It will be a copy-paste of the file with only difference being the `kubectl` version when we need to bump it. Instead, we could put the `kubectl` version in the image tag to make different versions available (e.g., `docker.io/paritytech/kubetools-kubectl-1.22`)

Reply:
Another option is properly separate them in a directory under kubetools. It will look like this:

```
kubetools/
|--- kubectl-1.22/
|-- helm3/
```